app/src/info/CooperativeAddOnItemRequirementsGenerator.ts

```diff
@@ -148,22 +140,24 @@ export default class AddOnItemRequirementsGenerator implements IProjectInfoItemG
             let baNameBreak = aName.split(".");
 
             if (baNameBreak.length < 2 || baNameBreak[0] !== "animation") {
+              // CADDONIREQ110
               items.push(
                 new ProjectInfoItem(
                   InfoItemType.testCompleteFail,
                   this.id,
-                  110,
+                  CooperativeAddOnItemRequirementsGeneratorTest.BehaviorAnimationIdNotInExpectedForm,
                   `Behavior animation identifier is not in expected form of animation.xyz.animation_name`,
                   projectItem,
                   aName
                 )
               );
-            } else if (!AddOnRequirementsGenerator.isNamespacedString(baNameBreak[1])) {
+            } else if (!CooperativeAddOnRequirementsGenerator.isNamespacedString(baNameBreak[1])) {
+              // CADDONIREQ111
               items.push(
                 new ProjectInfoItem(
                   InfoItemType.testCompleteFail,
                   this.id,
-                  111,
+                  CooperativeAddOnItemRequirementsGeneratorTest.BehaviorAnimationControllerNameNotInExpectedForm,
```

Comment:
I believe this should be `BehaviorAnimationNameNotInExpectedForm` not `BehaviorAnimationControllerNameNotInExpectedForm`